src/lib/types.ts

```diff
@@ -108,6 +108,31 @@ export type CCIPMessage<V extends CCIPVersion = CCIPVersion> = V extends
       tokenAmounts: readonly (EVM2AnyMessageSent['tokenAmounts'][number] & SourceTokenData)[]
     }
 
+type AptosTokenTransfer = {
+  sourcePoolAddress: string
+  destTokenAddress: string
+  destGasAmount: bigint
+  extraData: string
+  amount: bigint
+}
+// TODO Rodrigo: Refactor it to be included in CCIPMessage type
+export type AptosCCIPMessage = {
```

Comment:
This type should be compatible with `CCIPMessage<V1_6>`, no need to have a specific one for Aptos; you can use `decodeMessage` to juggle properties around to be compatible here, if needed